.travis.yml

```diff
@@ -117,6 +117,15 @@ script:
     else
       make -j$(nproc) check
     fi
+# check fuzz targets
```

Comment:
- This only runs them for a single iteration so make sure they compiled, linked, and run correctly
- I'll be submitting this to OSS Fuzz so that it does get run 24/7
- [ ] John will check out crypto backends to see how that works

Reply:
thanks for clarifying